### PR TITLE
add label color to status column

### PR DESCRIPTION
**Pull request by phyothiha**
## Issue Reference
#10387

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/views/sales/transactions/index.blade.php [hunk 1]
```diff
@@ -204,11 +204,23 @@ class="text-gray-600 dark:text-gray-300"
...
+                                        <p class="label-pending">
+                                            @lang('admin::app.sales.transactions.index.datagrid.pending')
+                                        </p>
+                                    </template>
+
+                                    <template v-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.completed')'">
+                                        <p class="label-completed">
+                                            @lang('admin::app.sales.transactions.index.datagrid.completed')
+                                        </p>
+                                    </template>
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/views/sales/transactions/index.blade.php:223]: We can use `v-else-if` here to optimize the code slightly, and the v-if condition can be added directly to the `<p>` tag for better readability and efficiency.

**Review by amit-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Resources/views/sales/transactions/index.blade.php [hunk 2]
```diff
@@ -204,10 +204,16 @@ class="text-gray-600 dark:text-gray-300"
...
+                                    <p v-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.paid')'" class="label-active">
+                                        @lang('admin::app.sales.transactions.index.datagrid.paid')
+                                    </p>
+
+                                    <p v-else-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.pending')'" class="label-pending">
+                                        @lang('admin::app.sales.transactions.index.datagrid.pending')
+                                    </p>
+
+                                    <p v-else-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.completed')'" class="label-completed">
+                                        @lang('admin::app.sales.transactions.index.datagrid.completed')
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/views/sales/transactions/index.blade.php:216]: When a tag contains two or more attributes, we should separate each attribute onto a new line for better readability.

You can refer to the example code snippet below:

```
<p
    v-if="data.invoice_id"
    class="text-gray-600 dark:text-gray-300"
    v-text="data.invoice_id"
>
</p>
```

**Review by amit-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Resources/views/sales/transactions/index.blade.php [hunk 3]
```diff
@@ -204,15 +204,24 @@ class="text-gray-600 dark:text-gray-300"
                                     </p>
 
 
-                                    <p v-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.paid')'" class="label-active">
+                                    <p
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/views/sales/transactions/index.blade.php:207]: Could you please remove the extra space before the <p> tag?

**Review by amit-webkul — CHANGES_REQUESTED** [review]

**Comment by devansh-webkul** [comment]
We will do this internally.

**Pull request closed without merging**